### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by tedsuo**
This pull request versions the OpenTelemetry specification to v1.0.

If there are any open PRs or issues which you believe would block this merge, please list them here. 

Remaining issues marked as required for tracing GA can be found here: https://github.com/open-telemetry/opentelemetry-specification/issues?q=is%3Aopen+label%3Arelease%3Arequired-for-ga+-label%3Aspec%3Ametrics+label%3Apriority%3Ap1

### CODE REVIEW

**Comment by Oberon00** [comment]
Before we release 1.0 don't we need to feature-freeze tracing and resource SDKs at least?
Also, I think we need to put something like "Document status: working draft" into non-GA specification, to avoid misrepresentation as GA.

**Comment by tedsuo** [comment]
Tracing and resources don't need to be frozen in general, but future changes will need to be backwards compatible once components are marked as stable. Are there breaking changes coming to those components? I didn't see them in the spec project. 

I agree that we should clearly and consistently mark every document in the spec as **stable** or **experimental**. 

I'm not sure that a separate branch would be helpful... keeping it in sync sounds annoying. I think we need  to educate spec readers (and users in general) about the singal lifecycle and get people comfortable checking components for their level of stability.

**Comment by carlosalberto** [comment]
In theory we need to get the issues that have been triaged as Required, minus the Metrics portion: https://github.com/open-telemetry/opentelemetry-specification/issues?q=is%3Aopen+is%3Aissue+label%3Arelease%3Arequired-for-ga+-label%3Aspec%3Ametrics+

An additional question is whether we need, for this specific release, the OT compatibility part (which is on the works although, hopefully, very close to be completed).

**Comment by tedsuo** [comment]
Thanks @carlosalberto I updated the links.

IMHO, OT and OC compatibility can be marked as optional for v1.0, as adding them is not a backwards incompatible change. There has been almost no work on the OC front, so it is not a reasonable requirement.

**Comment by yurishkuro** [comment]
> IMHO, OT and OC compatibility can be marked as optional for v1.0, as adding them is not a backwards incompatible change.

Except that it has the bad optics of being the 15th standard. Compatibility was explicitly called out as one of the success criteria of the new project.

**Comment by carlosalberto** [comment]
@yurishkuro My hope is that this week or the next week we finish the OT part (which, I'd imagine, is a MUST by the time Metrics is *also* available). Agreed that otherwise we would have missed a point.

For the record: the OT compatibility part is a high priority item for me in the following weeks ;)

**Comment by tedsuo** [comment]
@yurishkuro  I don't intend to say we are not backwards compatible, just that we don't need it required until both tracing and metrics are completed.

**Comment by tedsuo** [comment]
@Oberon00 Added lifecycle statuses to each document, and bumped the relevant documents to stable: https://github.com/open-telemetry/opentelemetry-specification/pull/1385

**Comment by jsuereth** [comment]
@yurishkuro Regarding OT=>OC compatibility (and vice versa), you can see the pending specifciation #1332  in place now.  For the purpose of 1.0 of Trace, we think that specification represents the best-compatibility guarantees we can provide OT<=>OC compatibility.   While there are some details that diverge, it's compatible in the 80% case.

My team has been working on OC=>OT compatibility.   Metrics stability has been a major blocker to anything concrete, but there should be working (and we can keep them stable) shims for Go + Java with Trace + 1.0.0.  We are (currently) looking into ripping the shim apart into two components (without requring another OC release).

In any case, I think the real point here is that we agree Trace 1.0.0 is ready as-is and we *can* provide the OT<=>OC bridge we need.  PTAL at the proposal or the Go/Java implementations and comment on #1332 if you have concerns.

**Comment by yurishkuro** [comment]
@jsuereth I think you are using OT to mean OpenTelemetry (please don't, the agreed abbreviation is OTEL/OTel, whereas OT stands for OpenTracing), so if you're asking about OTEL-OC interop then I am not the right person to ask since I never used OC.

**Comment by jsuereth** [comment]
@yurishkuro Sorry!, at work everyone refers to OTel as OT, will fix that in the future.  My point was more about whether OT=>Otel/OC=>Otel compatibility could be spec'd.  There's also an OT<=>OTel spec in progress.   I'm not aware of the details, but are we concerned that we wouldn't be able to implement a bridge?

**Comment by yurishkuro** [comment]
@carlosalberto was working on the OT bridge and spec.

**Comment by alolita** [comment]
My understanding for the official abbrevation is also OTEL or OTel as @yurishkuro pointed out.

@jsuereth please see https://github.com/open-telemetry/community/blob/main/marketing-guidelines.md 

Do you think there is a need for a project glossary? There are a lot of abbreviations used in the project docs and maintainer speak. The context can be confusing for contributors joining the project.

**Comment by cijothomas** [comment]
> My understanding for the official abbrevation is also OTEL or OTel as @yurishkuro pointed out.
> 
> @jsuereth please see https://github.com/open-telemetry/community/blob/main/marketing-guidelines.md
> 
> Do you think there is a need for a project glossary? There are a lot of abbreviations used in the project docs and maintainer speak. The context can be confusing for contributors joining the project.

https://github.com/open-telemetry/opentelemetry-specification#acronym

**Comment by alolita** [comment]
thanks @cijothomas

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ release.
 
 ## Unreleased
 
+## v1.0.0 (12-22-2021)
```
Comment (cijothomas) [inline CHANGELOG.md:10]: update date :)

**Review by cijothomas — APPROVED** [review]

**Review by lzchen — APPROVED** [review]

**Review by reyang — APPROVED** [review]
LGTM assuming the release date in CHANGELOG will be updated.

**Review by andrewhsu — APPROVED** [review]
LGTM

**Review by carlosalberto — APPROVED** [review]

**Comment by carlosalberto** [comment]
@open-telemetry/specs-approvers Please review so we can proceed 🍾

**Review by jsuereth — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Comment by bogdandrutu** [comment]
@open-telemetry/technical-committee should we wait until Monday or I can press the button today (since the PR is assign to me I am the lucky one to press the button)?

**Review by tigrannajaryan — APPROVED** [review]
Please fix the date.

**Comment by tigrannajaryan** [comment]
Can be merged once the date is corrected.

**Comment by bogdandrutu** [comment]
@andrewhsu Thank you for making this a reality, without your help we would not be here now :)

**Comment by bogdandrutu** [comment]
Moved the date to Monday since @open-telemetry/technical-committee did not approve it (we need one more to have at lest half of the members +1)

**Review by SergeyKanzhelev — APPROVED** [review]

**Review by carlosalberto — APPROVED** [review]

**Review by lalitb — APPROVED** [review]

**Comment by bogdandrutu** [comment]
I will wait few more hours then merge this.

**Comment by Oberon00** [comment]
Sorry to raise two more (related) potential blockers:
* I think we'll want to decide on / merge #1406 first
* https://github.com/anuraaga/opentelemetry-specification/blob/ottrace/specification/sdk-environment-variables.md is lacking a document/lifecycle status (was it missed in #1385?)

**Comment by carlosalberto** [comment]
> I think we'll want to decide on / merge #1406 first

Merged it as this is merely a follow up (and I guess I can make the call, given it's identifier related to OpenTracing ;) )

> https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md is lacking a document/lifecycle status (was it missed in #1385?)

Oh yes, I had already mentioned it to @tedsuo . Let's do a tiny follow up to do that, and we can release 🎆

**Review by arminru — APPROVED** [review]

**Comment by tedsuo** [comment]
@Oberon00 @carlosalberto added stable to ENV VARS: https://github.com/open-telemetry/opentelemetry-specification/pull/1409

**Comment by dyladan** [comment]
The trace id ratio based sampler still has "TODO: Add details about how the TraceIdRatioBased is implemented as a function of the TraceID."

Since this is our _only_ sampler that isn't always on or always off, I feel like it should be properly specified before 1.0

**Comment by Oberon00** [comment]
I created #1412 with a suggestion on how to resolve this for now.

**Comment by tedsuo** [comment]
Almost There

**Review by bruno-garcia — APPROVED** [review]

**Review by dashed — APPROVED** [review]

**Comment by bogdandrutu** [comment]
@open-telemetry/technical-committee are you aware of any blocking issue for this? Otherwise in couple of hours I will do the release

**Review by jmacd — APPROVED** [review]

**Pull request merged**